### DISCUSSION TITLE
Little Symbols

### DISCUSSION THREAD

**Discussion by miszapatos** [OpenOrienteering/mapper#1844] [Questions & Answers]
I've just installed 0.9.4 version and I think it's running correctly, but I have the problem with the symbols, they are very little and I don't know who changes them.

Could you help me?

Thanks
![symbols](https://user-images.githubusercontent.com/6623916/103539356-2fdde980-4e98-11eb-9bd0-1f5c4925a368.jpg)

**Comment by mlerjen** [comment]
Try this:
![image](https://user-images.githubusercontent.com/15611653/103539725-d75b1c00-4e98-11eb-8b78-8417fd9c9bb4.png)

**Reply by miszapatos** [reply]
Thanks,
Sorry for the question but I've not found a guide and Ididn't know where change it.

**Reply by ghost** [reply]
> but I've not found a guide

Actually "Mapper Manual" not include such info yet.

Guess, we need add more info into "Settings" chapter according latest Mapper version:

- https://www.openorienteering.org/mapper-manual/pages/settings.html
